.github/workflows/new-app-review/validators/boilerplate.ts

```diff
@@ -0,0 +1,32 @@
+import type { PullRequestFile, ValidatorOptions, ValidatorResult } from '../../types';
+
+
+export const validate = async (_options: ValidatorOptions, newAppDir: string, files: PullRequestFile[]): Promise<ValidatorResult> => {
+  const hasInvalidBoilerPlate = files
+    .filter((file) => file.status === 'added' && (file.filename.endsWith('.ts') || file.filename.endsWith('.tsx')))
+    .some((file) => {
+
+      if (file.patch) {
+        const paragraphComponentRegex = /<Paragraph[^>]*>([\s\S]*?)<\/Paragraph>/i;
+        const paragraphComponent = file.patch.match(paragraphComponentRegex);
+
+        if (paragraphComponent) {
+          const innerContent = paragraphComponent[1];
+          const helloRegex = /Hello/i;
+          return helloRegex.test(innerContent);
```

Comment:
I noticed the ConfigScreen boilerplate component has some slightly different wording, instead of "Hello {some} Component" it says "Welcome to your contentful app. This is your config page." [shown here](https://github.com/contentful/create-contentful-app-examples/blob/main/v2/templates/alpha-typescript/src/locations/ConfigScreen.tsx#L63)

Should we add another regex check here to look for this string too?